### ISSUE TITLE
80244008: Unknown error

### ISSUE THREAD

**Issue by andika207** [LegacyUpdate/LegacyUpdate#221]
Legacy Update does not work in this XP partition, even so the Windows Update yellow shield keeps showing off.

this partition was created back in 2020 when I had another WSUS patch installed which doesn't work anymore.

I checked the registry entry and it appears to be correct WUStatusServer... http://legacyupdate.net/v6

while checking the KB numbers I realized some updates have never been rolled out by Legacy Update by default
 **KB2487367 - KB2668562 - KB2636927 - KB969084 - KB2264107**.....

![screenshot 17](https://github.com/LegacyUpdate/LegacyUpdate/assets/47242117/9ce0ca49-e6e5-493e-864c-7321dab84c4e)

**Comment by andika207** [comment]
this post of mine got to my mind. https://github.com/LegacyUpdate/LegacyUpdate/issues/195#issuecomment-1880508549
I noticed two identical KB numbers which got me suspecting something strange was going on
then I checked the last updates that I had recently installed with WU MiniTool and all failed to install. (**0x80240034**)

so the big question is.... how or why is Windows Update detecting non installed updates if the WSUS server on the other side is OFF.

**Comment by NarakuITA** [comment]
Hi, I have the same problem, is there a way to solve it. I installed Windows XP SP3 on TOSHIBA Satellite Pro 4600 and then activated Windows, via phone. After that I installed the Legacy Update. Once done I looked for the updates and installed 141 of them, leaving out the optional ones.
Once all the updates were installed and the PC restarted, I tried to search for the remaining ones again, the optional ones, but to my great surprise, once the check was finished this error appeared which prevented me from downloading the remaining ones.
How can I solve this problem?

**Comment by metafarion** [comment]
Yeah, I'm getting this as well immediately after installing Legacy Update.  Is there a log file we should be looking in to try and get more details?

**Comment by NarakuITA** [comment]
In the end I solved it by reinstalling Windows XP

**Comment by andika207** [comment]
> I'm getting this as well immediately after installing Legacy Update.

I have installed Legacy Update countless times and download everything with no issues
this is actually a specific issue to this XP partition that was patched before Legacy Update ever existed

**Comment by NarakuITA** [comment]
I installed windows xp embedded posready 2009 and solved the root problem. I've downloaded all the PosReady 2009 updates.

**Comment by Sm0keySa1m0n** [comment]
I'm getting this on Windows 2000 too.

**Comment by temerkhanov** [comment]
Observed on Windows 2000 Professional SP4, but not on Advanced Server SP4
